internal/fptest/networkinstance.go

```diff
@@ -43,3 +45,29 @@ func AssignToNetworkInstance(t testing.TB, d *ondatra.DUTDevice, i string, ni st
 		gnmi.Update(t, d, gnmi.OC().NetworkInstance(ni).Config(), netInst)
 	}
 }
+
+// EnableGribiOnNetworkInstance enables Gribi protocol under network instance.
+func EnableGribiUnderNetworkInstance(t testing.TB, d *ondatra.DUTDevice, ni string) {
+	t.Helper()
+	if ni == "" {
+		t.Fatalf("Network instance not provided for protocol definition")
+	}
+	cliCmd := fmt.Sprintf("/set network-instance %s protocols gribi admin-state enable", ni)
```

Comment:
@trathod1 since this is a very Nokia CLI specific, do you mind do a pre condition check here? e.g. `if tc.vendor != "Nokia", t.Fatlf ()`

Reply:
@xw-g I'll add this in the commit for the migration from CLI origin to Native model.

Reply:
Added the safeguard, I decided to do a no-op instead of `t.Fatalf`. This way the code won't be executed by other vendors and the test won't fail if someone mistakenly uses this deviation